data/blog/2024/2024-07-23-current-maintenance-status-of-laminas-mezzio-packages.md

```diff
@@ -0,0 +1,36 @@
+---
+id: 2024-07-23-current-maintenance-status-of-laminas-mezzio-packages
+author: julian
+title: 'Current Maintenance Status of Laminas & Mezzio Packages'
+draft: false
+public: true
+created: '2024-07-23T11:00:00-01:00'
+updated: '2024-07-23T11:00:00-01:00'
+tags:
+  - custom proprieties
+  - maintenance status
+---
+
+The Laminas Project has created a large number of packages to serve the needs of the PHP community.
+But what happens when you need to make sure that the package you want to use in your application is being actively
+maintained?
+Sure, you can check each package manually, but the most useful thing would be a full overview of this vital
+information.
+
+<!--- EXTENDED -->
+
+### Laminas and Mezzio packages maintenance status at a glance
+
+The page below intends to provide exactly that - a fast, accessible way to examine every package at a glance. It
+contains columns for each of the three organizations: Laminas, Mezzio and Api Tools. The page automatically refreshes
+the package status daily, is sorted alphabetically and is publicly available.
+
+[**Current Maintenance Status of Laminas & Mezzio Packages**](https://getlaminas.org/packages-maintenance-status/)
+
+### Stay vigilant!
+
+Things are constantly in motion, though, so you need to check on this page regularly to make sure that the packages you
+chose for your project are still reliable. If the status of a package changes from active to `maintenance-only`,
+`security-only` or is `discontinued`, you can quickly see it and take action.
+The handy link to the relevant Technical Steering Committee minute file details the reason for the status change and a
+recommended solution for moving forward.
```

Comment:
This part sounds rather negative to me: "Stay vigilant!", "…you need to check…" and "…are still reliable…". We make it mandatory for the user, otherwise he has lost. 
Either we describe the advantages of the simple and up-to-date overview, or the section has to be deleted completely.

Another option is to simply wait until the documentation for the individual packages also contains the information, then we can offer several places to read the status.

Reply:
I just updated the article